File: code/Calculator.java

public class Calculator {

    // Static method
    public static int sum(int a, int b) {
        return a + b;
    }

    public static void main(String[] args) {
        // Calling a static method without creating an object
        int result = Calculator.sum(8, 2);
        System.out.println("The sum is: " + result);
    }
}

File: code/Calculator.scala

object Calculator {
  // Method equivalent to a Java static method
  def sum(a: Int, b: Int): Int = {
    a + b
  }

def main(args: Array[String]): Unit = {
  // Call method sum without create a object
  val result = Calculator.sum(2, 11)
  println("The sum is: " + result)
  }
}


